components/eam/bld/build-namelist

```diff
@@ -3567,13 +3567,13 @@ add_default($nl, 'use_subcol_microp');
 add_default($nl, 'microp_scheme');
 add_default($nl, 'micro_do_icesupersat');
 add_default($nl, 'macrop_scheme');
+add_default($nl, 'nucleate_ice_subgrid');
+add_default($nl, 'so4_sz_thresh_icenuc');
+add_default($nl, 'cld_macmic_num_steps');
```

Comment:
These changes cause the above three namelist variables to appear for non-conventional atm configurations.

Reply:
No need to commit any adjustments for now.